bundles/org.openhab.binding.metofficedatahub/README.md

```diff
@@ -36,10 +36,11 @@ This binding supports the follow thing types:
 The bridge counts the total number of requests from 00:00 -> 23:59 under its properties during the runtime of the system.
 (This reset's if OH restarts, or the binding resets).
 
-| Name               | Type   | Description                                                                               | Default Values |
-|--------------------|--------|-------------------------------------------------------------------------------------------|----------------|
-| siteRateDailyLimit | Number | This is a daily poll limit for the SiteSpecific API, while the Thing ID remains the same. | 250            |
-| siteApiKey         | String | The API Key for the Site Specific subscription in your MET Office Data Hub account.       |                |
+| Name                 | Type    | Description                                                                               | Default Values |
+|----------------------|---------|-------------------------------------------------------------------------------------------|----------------|
+| siteRateDailyLimit   | Number  | This is a daily poll limit for the SiteSpecific API, while the Thing ID remains the same. | 250            |
+| siteApiKey           | String  | The API Key for the Site Specific subscription in your MET Office Data Hub account.       |                |
+| disableKeyValidation | Boolean | If enabled the binding will not attempt to validate a valid key format has been entered.  | false          |
```

Comment:
Maybe change it to be more readable. 

```suggestion
| disableKeyValidation | Boolean | If enabled, the binding will skip validation of the key format.  | false          |
```

Reply:
yeah I like that better will update that in the readme and the description of the config item.

Reply:
Oh, i did not properly align the whitespaces. 
(and properties file)

Reply:
Thank you @lsiepel - also sorry is it the 5.0.X branch for backporting. Are we good to back port the new setting as well?

Reply:
ah bleep that's a silly mistake to forget thanks running now

Reply:
just pushed thanks :) (Need to get that one added to copilots auto-checks :)